golang_test.go

```diff
@@ -57,7 +60,7 @@ func TestGolangApplier_CheckHeader(t *testing.T) {
 			if err != nil {
 				t.Fatalf("failed to check header: %+v", err)
 			}
-			expected := strings.HasSuffix(fileName, ".golden")
+			expected := strings.HasSuffix(fileName, ".golden") || strings.HasSuffix(fileName, ".generated")
```

Comment:
".generated.golden" || ".generated"

Reply:
I think I'm missing one of the other files because of that; let me change this test to have an explicit option that tells if it should match or not.